docker/images/skycoindev/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# skycoin build
+# reference https://github.com/skycoin/skycoin
+FROM golang:1.10.0-stretch
+
+# dirs
+RUN apt-get update && apt-get install -y cmake gdbserver gdb vim ctags vim-scripts screen
+RUN git clone --recurse-submodules -j8 https://github.com/Snaipe/Criterion
```

Comment:
Do not clone this here . Criterion has nothing to do with go lang . Maybe it should be built in a separate image and library binaries plus headers copied in expected locations ?